images/ubuntu/19.04/extra-packages

```diff
@@ -0,0 +1,18 @@
+bash-completion
+git
+keyutils
+libcap2-bin
+lsof
+man-db
+mlocate
+mtr
+packagekit-command-not-found
```

Comment:
`packagekit-command-not-found` shouldn't be included. See commit 0dee71b13f71fca9a37c1595f73a71727cafb67f

Reply:
Agreed, it doesn't work anyway.